File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        n = len(nums)
        majority = n//3
        hash_map = {}
        result = []

        for i in range(len(nums)):
            if nums[i] not in hash_map:
                hash_map[nums[i]] = 1
            else:
                hash_map[nums[i]] += 1

        for num, freq in hash_map.items():
            if freq > majority:
                result.append(num)

        return result        


        

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.cpp

class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int result = 0;
        int counter = 0;

        for(int i = 0; i <nums.size(); i++){
            if(nums[i] != 1){
                counter = 0;
            }
            else{
                counter ++;
                result = max(result, counter);
            }

        }
        return result;
        
    }
};